content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -40,81 +40,19 @@ replaces the functionality provided by the `istio-init` container.
     *  The Kubernetes documentation highly recommends this for all Kubernetes installations
        where `ServiceAccounts` are utilized.
 
-1.  [Download the Istio release](/docs/setup/kubernetes/#downloading-the-release).
```

Comment:
The Istio installation needs to be part of the prerequisites. Readers need to be able to enable CNI independently from installing Istio.

Reply:
No, it can't be done before installing Istio CNI.
The installation of Istio and the Istio CNI is done in the 2nd step of the "Installation" section.